app/stac_api/validators_serializer.py

```diff
@@ -246,20 +245,6 @@ def validate(self, request):
         '''
         # harmonize GET and POST
         query_param = harmonize_post_get_for_search(request)
```

Comment:
Pay attention, the `harmonize_post_get_for_search()` method returns the query either from the URL query or from the POST data ! Not sure it is a good idea to also remove this check in the POST data ? What would happen if we provide a POST with invalid attributes ?